crates/eip7928/src/block_access_list.rs

```diff
@@ -0,0 +1,13 @@
+//! Contains the [`BlockAccessList`] type, which represents a simple list of account changes.
+
+use crate::account_changes::AccountChanges;
+use alloc::vec::Vec;
+
+/// This struct is used to store `account_changes` in a block.
+pub type BlockAccessList = Vec<AccountChanges>;
```

Comment:
I see, we can keep this as type alias for now, and maybe introduce a new type for this if useful